src/officehours_ui/templates/base.html

```diff
@@ -21,94 +21,121 @@
+                  <a class="nav-link dropdown-toggle" href="#" id="userMenu" role="button" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    {{ request.user.username }}
+                  </a>
+                  <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="userMenu">
+                    <li>
+                      <a class="dropdown-item" href="{% url 'manage' %}" style="background-color: white; color:#212529">
+                        Manage Queues
+                      </a>
+                    </li>
+                    <li>
+                      <a class="dropdown-item" href="{% url 'preferences' %}" style="background-color: white; color:#212529">
+                        Preferences
+                      </a>
+                    </li>
+                    <li>
+                      <form action="{% url 'oidc_logout' %}" method="post">
+                        {% csrf_token %}
+                        <input type="submit" value="Logout" style="background-color: white" class="dropdown-item" id="logout-button">
+                      </form>
+                    </li>
+                  </ul>
+                </li>
+              {% else %}
+                <li class="nav-item">
+                  <a href="{{ LOGIN_URL }}" class="nav-link">Login</a>
+                </li>
+              {% endif %}
+            </ul>
+          </div>
         </div>
-      </div>
-    </nav>
-  </header>
+      </nav>
+    </header>
+
+
+    {% for page in flatpages %}
+      {% if page.title == "User Comm Banner" %}
+        {{page.content|safe}}
+      {% endif %}
+    {% endfor %}
 
-  {% for page in flatpages %}
-    {% if page.title == "User Comm Banner" %}
-      {{page.content|safe}}
-    {% endif %}
-  {% endfor %}
 
-  <div id="main-content" class="container" role="main">
-    {% block content %}{% endblock %}
-    {% csrf_token %}
-	</div>
+    <div id="main-content" class="container" role="main">
+      {% block content %}{% endblock %}
+      {% csrf_token %}
+    </div>
 
-  <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.6/dist/umd/popper.min.js" integrity="sha384-oBqDVmMz9ATKxIep9tiCxS/Z9fNfEXiDAYTujMAeBAsjFuCZSmKbSSUnQlmh/jp3" crossorigin="anonymous"></script>
-  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.min.js" integrity="sha384-cuYeSxntonz0PPNlHhBs68uyIAVpIIOZZ5JqeqvYYIcEL727kskC66kF92t6Xl2V" crossorigin="anonymous"></script>
-  {% block scripts %}{% endblock %}
 
-  {% get_flatpages '/dataalert/' as flatpages %}
-  {% if flatpages.first and flatpages.first.content %}
-    <footer class="sticky-footer">
-      <div class="alert alert-info alert-footer" role="alert">
-        <div class="alert-footer-text">
-          {{flatpages.first.content|safe}}
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
```

Comment:
@jonespm has maybe mentioned this before, but I'm not seeing anywhere in the code that actually makes use of the JQuery import. If everything behaves as intended without this, I suggest removing it.

Reply:
Appears to work without JQuery so I removed it